docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1687,3 +1687,36 @@ The connector supports redirection from Iceberg tables to Hive tables with the
 
 The connector supports configuring and using [file system
 caching](/object-storage/file-system-cache).
+
+### Manifest File Caching
+
+As of Iceberg version 1.1.0, Apache Iceberg provides a mechanism to cache the
+contents of Iceberg manifest files in memory. This feature helps to reduce
+repeated reads of small Iceberg manifest files from remote storage.
+
+Note that currently, manifest file caching is supported for Hive Metastore, Glue,
+JDBC, Nessie, Rest and Snowflake catalogs in the Iceberg connector.
```

Comment:
If the feature requires a catalog specific code, it should not be configured in IcebergConfig, but rather a separate (new) config class that catalog modules will bind or not, depending whether they support given feature.

this prevents a situation where config is bound & available, but not used, doesn't mean anything.

Reply:
The feature is integrated within the Iceberg library, we just need to pass the configurations to the Iceberg api, so you are right, it isn't a catalog specific code, i will remove this description~

Reply:
the plumbing is catalog-specific, afaict from reading the code

Reply:
> the plumbing is catalog-specific, afaict from reading the code

You are right, after reading the iceberg codes: https://github.com/apache/iceberg/pull/4518/files  
I found that it only support `HadoopFileIO`